### PR TITLE
validate isRdma config in operator webhook

### PR DESCRIPTION
**Pull request by zshi-redhat**

### CODE REVIEW

File: pkg/webhook/validate.go [hunk 1]
```diff
@@ -106,6 +106,15 @@ func staticValidateSriovNetworkNodePolicy(cr *sriovnetworkv1.SriovNetworkNodePol
 			}
 		}
 	}
+
+	// TODO: Add automatic correction in Operator mutation webhook
```
Comment (pliurh) [inline pkg/webhook/validate.go:110]: Normally, an admission controller doesn't do the automatic correction. I prefer we only set the default value if a flag is not set.

Comment (zshi-redhat) [reply pkg/webhook/validate.go]: removed.

This is related to one idea that we do intelligent analysis, we can discuss it later.

**Comment by pliurh** [comment]
Could you also add a test to `pkg/webhook/validate_test.go`?

**Comment by zshi-redhat** [comment]
> Could you also add a test to `pkg/webhook/validate_test.go`?

added

**Comment by pliurh** [comment]
v

**Pull request merged**